tests/test_webclient.py

```diff
@@ -72,14 +78,16 @@ def test_externalUnicodeInterference(self):
         elements of its return tuple, even when passed an URL which has
         previously been passed to L{urlparse} as a C{unicode} string.
         """
-        badInput = u'http://example.com/path'
-        goodInput = badInput.encode('ascii')
+        goodInput = u'http://example.com/path'
+        badInput = goodInput.encode('ascii')
+        if six.PY2:
+            goodInput, badInput = badInput, goodInput
```

Comment:
This looks a bit odd to me -- why is this inverted in PY2 and PY3?
I'm actually trying to understand the purpose of this test (btw, the docstring needs update).